com.unity.multiplayer.mlapi/Runtime/Core/NetworkTickSystem.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using UnityEngine;
+
+namespace MLAPI
+{
+    public class NetworkTickSystem : INetworkUpdateSystem, IDisposable
+    {
+        private const double k_DefaultTickDuration = 0.016;
+        private double m_TickDuration;
+        private int m_NetworkTick; //How many network ticks have passed?
```

Comment:
```suggestion
        private float m_TickInternalMs;
        private int m_NetworkTickCount;
```

I feel a little bit mixed here. if variable name is not good enough, we should probably change its name rather than comment next to it IMHO. I'd propose `m_NetworkTickCount` instead of `m_NetworkTick` and `m_TickIntervalMs` instead of `m_TickDuration` here. what do you think?

Reply:
Yeah, will change. I was trying to stay aligned with original branch, but it's worth improving upon.

Reply:
I would recommend using doubles for time tracking precision, but it is not something to stop this commit by any stretch of the imagination.

Reply:
Noted. I think the long-term goal will be:
Read float `unscaledtime`, because that's the precision we have.
Maintain an integral number of millis for precision alignment and then surface out any time via APIs in `double`